md/set-up-continuous-integration.md

```diff
@@ -9,7 +9,7 @@ Learn how to set up a continuous integration (CI) environment for your Bonita pr
 ::: warning
 **Note:**
 * The solution described here relies on the BonitaStudioBuilder, which has been deprecated since Bonita 7.7.0. Instead, we strongly encourage you to use the LA builder included in the tooling suite of Bonita Continuous Delivery add-on. One added-value is that LA builder does not need a studio to be installed.
-* [Bonita Continuous Delivery add-on](https://documentation.bonitasoft.com/bcd/2.0/) comes with an out-of-the-box solution based on Jenkins CI.
+* [Bonita Continuous Delivery add-on](https://documentation.bonitasoft.com/bcd/3.3/) comes with an out-of-the-box solution based on Jenkins CI.
```

Comment:
I think you should use https://documentation.bonitasoft.com/bcd/current/ in the link, to ensure to always point to the latest version, even when BCD 3.4 is out.
@tnguyen1 do you confirm?

Reply:
With current documentation website indeed, you may use the https://documentation.bonitasoft.com/bcd/any-string-you-like/ pattern to link to the latest version of the component.
I saw here https://github.com/bonitasoft/bonita-doc/blame/2f279f1d1bbbf80a881708cffcfe6879a911f148/md/automating-builds.md#L8 that the https://documentation.bonitasoft.com/bcd/latest/ URL is used instead.